### PR TITLE
GEOMESA-1835 Fixing cleanup of shapefile ingest

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
the `TypeUpdatingFeatureCollection` seems like it was implemented properly - instead of trying to fix it for a general case, I just moved the type updating logic into the shapefile ingest (as that is the only place using it)

File: geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
                 <exclude>org.locationtech.geomesa:geomesa-geojson*</exclude>
                 <exclude>org.locationtech.geomesa:geomesa-blobstore*</exclude>
                 <exclude>org.locationtech.geomesa:geomesa-accumulo-compute*</exclude>
+                <exclude>org.locationtech.geomesa:geomesa-accumulo-spark-runtime*</exclude>
```
Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml]: +1!

Comment (aheyne) [inline geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml]: Need to update the docs here http://www.geomesa.org/documentation/current/user/spark/accumulo_spark_runtime.html?highlight=spark

Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml]: it's still in `dist/spark/`, just not under `lib/` - i think the docs are ok as is?

Comment (aheyne) [inline geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml]: Ah, yeah, good catch then!

**Pull request merged**